### PR TITLE
CB-66 IdAM cookie banner config changes

### PR DESCRIPTION
**Pull request by johnholland-nice**

### CODE REVIEW

File: src/__snapshots__/cookie-control-config.test.ts.snap [hunk 1]
```diff
@@ -24,7 +24,12 @@ Object {
     "__ngat_1.2",
     "__nrpa_2.2",
     "IdAM",
-    "IdAM-*",
+    "IdAMC*",
```
Comment (ediblecode) [inline src/__snapshots__/cookie-control-config.test.ts.snap:27]: Don't we want IdAM-* too?

**Review by ediblecode — CHANGES_REQUESTED** [review]

File: src/__snapshots__/cookie-control-config.test.ts.snap (hunk 1 above)
Comment (John-Williams) [reply src/__snapshots__/cookie-control-config.test.ts.snap:27]: i think i got the cookie wrong for this one and John's right. 

the cookie used by local/dev sites, which don't use redis is: IdAMC[x]

all the sites deployed to servers though just have a single idam cookie called IdAM, which just contains a session key, so this hasn't affected anything except dev machines not using redis.

**Comment by johnholland-nice** [comment]
https://nicedigital.atlassian.net/browse/CB-66

File: src/__snapshots__/cookie-control-config.test.ts.snap (hunk 1 above)
Comment (ediblecode) [reply src/__snapshots__/cookie-control-config.test.ts.snap:27]: Great, thanks for clarifying, will approve then!

**Review by ediblecode — DISMISSED** [review]

**Review by ediblecode — APPROVED** [review]

**Pull request merged**